src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx

```diff
@@ -129,9 +152,10 @@ export const ArtworkSidebarCurrentBidInfo: React.FC<ArtworkSidebarCurrentBidInfo
             </Box>
           )}
 
-          <Text variant="lg" pl={0.5}>
+          {/* force a different key on each re-render so animation triggers */}
+          <CurrentBid key={Math.random()} variant={"lg"} pl={0.5}>
```

Comment:
A new key forces a re-render of this element, which is to enable playing the animation once, _each time there is a re-render_. Without it, the animation plays once the first time this component re-renders with an updated current bid, but then subsequent times as the bid is increasing, it doesn't play.